### PR TITLE
Take

### LINKED ISSUE #19210: take

### PR DESCRIPTION
**Pull request by MuhammadNizamani**
Closed  #19210

### CODE REVIEW

**Comment by V-G-spec** [comment]
Hey @MuhammadNizamani, thanks for your contribution! However, I see branch conflict with your PR. Do you mind taking a look?

**Comment by MuhammadNizamani** [comment]
@V-G-spec sure

**Comment by V-G-spec** [comment]
@MuhammadNizamani thanks for resolving the conflicts. Can you take a look at the failing tests next?

**Comment by MuhammadNizamani** [comment]
@V-G-spec it is passing all tests locally.

**Comment by V-G-spec** [comment]
Can you go through the logs for failing tests, then? And also share a screenshot of the tests passing locally

**Comment by MuhammadNizamani** [comment]
> Can you go through the logs for failing tests, then? And also share a screenshot of the tests passing locally

sure

**Comment by MuhammadNizamani** [comment]
@V-G-spec  There is issue in method name  `test_jax_round`  in test-pr 20 and i have gone through all failed tests log and any of the error was not releteded to my changes. 
and in following screenshot 
take function test

**Comment by V-G-spec** [comment]
I can see errors in PR test 1 as well. 
```
ERROR: not found: /ivy/ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py::test_jax_devicearray_take
(no name '/ivy/ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py::test_jax_devicearray_take' in any of [<Module test_devicearray.py>])
```

**Comment by MuhammadNizamani** [comment]
@V-G-spec can you tell me could be the error that I can resolve because it is passing all tests.

**Comment by V-G-spec** [comment]
I'm not quite sure myself. If you go through the logs, you'll find some more details. It was something I have come across for the first time. Lmk if you can't figure it out. I'll raise it internally

**Comment by V-G-spec** [comment]
Btw, you can also ask kapa on Discord to help with the error

File: ivy/functional/frontends/jax/array.py [hunk 1]
```diff
@@ -273,6 +273,18 @@ def __iter__(self):
...
+        self,
+        indices,
+        axis=None,
+        out=None,
+        mode=None,
+        unique_indices=False,
+        indices_are_sorted=False,
+        fill_value=None,
+    ):
+        return jax_frontend.numpy.take(self, indices, axis=axis, out=out)
```
Comment (zhumakhan) [inline ivy/functional/frontends/jax/array.py:286]: Could please also handle `mode` argument in here

Comment (zhumakhan) [reply ivy/functional/frontends/jax/array.py:286]: Rest seems good!

**Review by zhumakhan — CHANGES_REQUESTED** [review]

**Comment by MuhammadNizamani** [comment]
@zhumakhan  Sir mode = None in the JAX documantion and the I am using `jax_frontend.numpy.take` .

**Comment by zhumakhan** [comment]
jax_frontend.numpy.takes seems to have mode argument: https://github.com/unifyai/ivy/blob/main/ivy/functional/frontends/jax/numpy/manipulations.py#L111

File: ivy/functional/frontends/jax/array.py (hunk 1 above)
Comment (zhumakhan) [inline ivy/functional/frontends/jax/array.py:286]: Please call `jax_frontend.numpy.take` with `mode` argument. E.g.: `jax_frontend.numpy.take(self, indices, axis=axis, out=out, mode=mode)`

**Review by zhumakhan — CHANGES_REQUESTED** [review]

**Comment by MuhammadNizamani** [comment]
@zhumakhan  review this please

**Pull request closed without merging**